libs/sdk-embedding/src/iframe/kd.ts

```diff
@@ -1023,4 +1039,131 @@ export namespace EmbeddedKpiDashboard {
     ): obj is OpenScheduleEmailDialogCommandData {
         return getEventType(obj) === GdcKdCommandType.OpenScheduleEmailDialog;
     }
+
+    /**
+     * @public
+     */
+    export interface ISetFilterParentsAttributeFilter {
+        attributeFilter: {
+            displayForm: ObjRef;
+        };
+    }
+
+    /**
+     * @public
+     */
+    export type SetFilterParentsItemFilter = ISetFilterParentsAttributeFilter;
+
+    /**
+     * @public
+     */
+    export interface ISetFilterParentsItemParent {
+        parent: SetFilterParentsItemFilter;
+        connectingAttribute: ObjRef;
+    }
+
+    /**
+     * @public
+     */
+    export interface ISetFilterParentsItem {
+        filter: SetFilterParentsItemFilter;
+        parents: ISetFilterParentsItemParent[];
+    }
+
+    /**
+     * @public
+     */
+    export interface ISetFilterParentsDataBody {
+        filters: ISetFilterParentsItem[];
+    }
```

Comment:
please look at these from consumer perspective and add TSDoc for the various types and properties. the documentation should give user enough information to know 'what to put where', how to construct the command body.

or look at it from perspective where we will be generating fancy public API documentation from the code. The API documentation should be verbose enough for consumers to understand how to construct the payload.